Integration/FRAIHMWORK Integrator Starter Kit Collection.json

```diff
@@ -0,0 +1,2038 @@
+														},
+														"url": {
+															"raw": "{{baseUrl}}{{Mitigation}}/subscription/mitigation/:uuid",
+															"host": [
+																"{{baseUrl}}{{Mitigation}}"
+															],
+															"path": [
+																"subscription",
+																"mitigation",
+																":uuid"
+															],
+															"variable": [
+																{
+																	"key": "uuid",
+																	"value": "{{UUIDmitigationSubscription}}"
+																}
+															]
+														},
+														"description": "Allows a system to report one or more faults currently affecting the system"
+													},
+													"response": []
+												},
+												{
+													"name": "Validate subscription details",
+													"event": [
+														{
+															"listen": "test",
+															"script": {
+																"exec": [
+																	"var jsonData = pm.response.json();\r",
+																	"\r",
+																	"pm.test(\"Returned Expected Value\", function () {\r",
+																	"    pm.expect(jsonData.interestedUuids[0]).to.eq(pm.collectionVariables.get(\"UUIDMitigationResponse\"));\r",
+																	"    pm.expect(jsonData.interestedUuids[1]).to.eq(pm.collectionVariables.get(\"UUIDMitigationResponse2\"));\r",
+																	"    pm.expect(new Date(jsonData.stopTime).valueOf()).to.eq(new Date(pm.collectionVariables.get(\"expirationTime\")).valueOf());\r",
+																	"});\r",
+																	"\r",
+																	"pm.test(\"Successful GET with UUID\", function () {\r",
+																	"    pm.response.to.have.status(200);\r",
+																	"});"
+																],
+																"type": "text/javascript"
+															}
+														}
+													],
+													"request": {
+														"auth": {
+															"type": "bearer",
+															"bearer": [
+																{
+																	"key": "token",
+																	"value": "{{access_token}}",
+																	"type": "string"
+																}
+															]
+														},
+														"method": "GET",
+														"header": [],
+														"url": {
+															"raw": "{{baseUrl}}{{Mitigation}}/subscription/mitigation/:uuid",
+															"host": [
+																"{{baseUrl}}{{Mitigation}}"
+															],
+															"path": [
+																"subscription",
+																"mitigation",
+																":uuid"
+															],
+															"variable": [
+																{
+																	"key": "uuid",
+																	"value": "{{UUIDmitigationSubscription}}"
+																}
+															]
+														}
+													},
+													"response": []
+												}
+											]
+										},
+										{
+											"name": "Cleanup",
+											"item": [
```

Comment:
the cleanup folder can be placed after the mitigation delete folder

Reply:
This cleanup is to demonstrate removing a subscription and moving the mitigation to a terminal state. It should not be at the same level of the Setup folder (at the top of the collection).